packages/module/patternfly-docs/content/extensions/chatbot/examples/Messages/MessageWithToolResponse.tsx

```diff
@@ -0,0 +1,141 @@
+import { useState, FunctionComponent, MouseEvent as ReactMouseEvent } from 'react';
+import Message from '@patternfly/chatbot/dist/dynamic/Message';
+import patternflyAvatar from './patternfly_avatar.jpg';
+import { CopyIcon, WrenchIcon } from '@patternfly/react-icons';
+import {
+  Button,
+  DescriptionList,
+  DescriptionListDescription,
+  DescriptionListGroup,
+  DescriptionListTerm,
+  ExpandableSection,
+  ExpandableSectionVariant,
+  Flex,
+  FlexItem,
+  Label
+} from '@patternfly/react-core';
+
+export const MessageWithToolResponseExample: FunctionComponent = () => {
+  const [isExpanded, setIsExpanded] = useState(false);
+
+  const onToggle = (_event: ReactMouseEvent, isExpanded: boolean) => {
+    setIsExpanded(isExpanded);
+  };
+
+  return (
+    <Message
+      name="Bot"
+      role="bot"
+      avatar={patternflyAvatar}
+      content="This example has a body description that's within the recommended limit of 2 lines:"
+      toolResponse={{
+        collapsedToggleText: 'Tool response: Name',
+        expandedToggleText: 'Tool response: Name',
+        subheading: 'Thought for 3 seconds',
+        body: 'Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.',
+        cardTitle: (
+          <Flex alignItems={{ default: 'alignItemsCenter' }} justifyContent={{ default: 'justifyContentSpaceBetween' }}>
+            <FlexItem>
+              <Flex direction={{ default: 'column' }} gap={{ default: 'gapXs' }}>
+                <FlexItem grow={{ default: 'grow' }}>
+                  <Flex gap={{ default: 'gapXs' }}>
+                    <FlexItem>
+                      <WrenchIcon style={{ color: 'var(--pf-t--global--icon--color--brand--default' }} />
+                    </FlexItem>
+                    <FlexItem>Tool name</FlexItem>
+                  </Flex>
+                </FlexItem>
+                <FlexItem>
+                  <Flex gap={{ default: 'gapSm' }} style={{ fontSize: '12px', fontWeight: '400' }}>
+                    <FlexItem>Execution time:</FlexItem>
+                    <FlexItem>0.12s</FlexItem>
```

Comment:
```suggestion
                    <FlexItem>0.12 seconds</FlexItem>
```
via our ux writing guidelines, we totally can abbreviate to "s", but since there's plenty of space to spell it out that would be better